src/mudlet.cpp

```diff
@@ -5182,6 +5192,30 @@ void mudlet::setInterfaceLanguage(const QString& languageCode)
     }
 }
 
+// return the user's desktop language if we have a quality translation for it
+// or a back-up language they've specified
+QString mudlet::autodetectPreferredLanguage()
+{
+    QVector<QString> availableQualityTranslations;
+    for (auto& code : getAvailableTranslationCodes()) {
+        auto& translation = mTranslationsMap.value(code);
+        if (translation.fromResourceFile()) {
+            auto& translatedPc = translation.getTranslatedPercentage();
+            if (translatedPc >= mTranslationGoldStar) {
```

Comment:
Come on, `mTranslationGoldStar` is currently 95% and last time I checked this covered only **four** languages:
* German (Germany)
* Italian (Italy)
* Russian (Russia)
* English (American)

This is probably not the best we can do: Chinese (Simplified) is at 90% and English (Great Britain) is only at 20% - BUT THE LATTER IS PERFECTLY USABLE AS I HAVE PICKED OUT MOST OF THE STRINGS THAT HAVE WORDS THAT *ARE* DIFFERENT BETWEEN IT AND AMERICAN.

At one stage I did posit a Silver Star for languages that were close to that Gold Star percentage (say 90 to 94%) and if we did something like that we could at least include the first of those two extra languages in the auto-selection ones. A manual addition of British as a special exception (because of the closeness to American) might help as well.

Reply:
If you say you'd like to English British English as an exception, that sounds good!

As for Chinese - only 5% to go and it'll be there!

No need to overcompilicate things with more categories, they can finish it up.

Reply:
Okay: Please can we have "the British English there as an exception"...

Reply:
Added.

Reply:
Could you finish up the translation? It doesn't look professional to have it at 20% and be preselected for their use. It's a minor detail, but if we can avoid it, why not :)